angular/src/app/admin/dam-repository/resources/resource-detail/resource-detail.component.ts

```diff
@@ -23,7 +24,8 @@ export class ResourceDetailComponent extends DamConfigEntityDetailComponentBase<
   @ViewChild(ResourceFormComponent, { static: false })
   resourceForm: ResourceFormComponent;
   @ViewChild('accessForm', { static: false })
-  accessForm: PersonaResourceAccessComponent;
+  accessForm: ResourceAccessComponent;
+  isDryRun: boolean;
```

Comment:
We can omit the field here. Instead you can use default parameter value on line https://github.com/DNAstack/ddap-frontend/pull/231/files#diff-2ce1c6cfb7d3ae253b223c451f1cec99R39 such as:
```
update(isDryRun = false)
```